packages/fuel-indexer-lib/src/utils.rs

```diff
@@ -69,8 +69,6 @@ pub struct IndexStopRequest {
 
 #[derive(Debug)]
 pub struct IndexRevertRequest {
```

Comment:
If we're renaming some types, we should probably rename this to `IndexerRevertRequest` and the others ones should also be renamed accordingly.